lib/entity.js

```diff
@@ -259,12 +259,15 @@ validateComponentType = function(componentType) {
   }
 };
 
-Entity.prototype[bSetup] = function(components) {
-  if (components && !(components instanceof Array)) {
-    throw new TypeError('expected array of components for entity');
-  }
-  if (components) {
-    fast.forEach(components, this.add, this);
+Entity.prototype[bSetup] = function(entityOrComponents) {
```

Comment:
Yeah, I reverted it in the src/, that was before we had `buildEntity`. I'll revert this as well. :)